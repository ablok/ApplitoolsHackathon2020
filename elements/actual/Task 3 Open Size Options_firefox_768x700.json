[
  {
    ".custom-select-form li": [
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 823.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 77, 218, 0.04)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "Not Small"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 855.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "M"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 887.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "L"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 919.0999755859376
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "XL"
      }
    ]
  }
]
