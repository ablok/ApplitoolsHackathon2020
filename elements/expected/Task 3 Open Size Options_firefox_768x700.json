[
  {
    ".custom-select-form li": [
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 822.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 77, 218, 0.04)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "Small (S)"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 854.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "M"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 886.0999755859375
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "L"
      },
      {
        "visibility": true,
        "location": {
          "x": 400,
          "y": 918.0999755859376
        },
        "size": {
          "width": 352,
          "height": 32
        },
        "css": {
          "display": "list-item",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": ""
        },
        "text": "XL"
      }
    ]
  }
]
